tinyengine/engine.py

```diff
@@ -64,7 +64,7 @@
 }
 
 
-def main():
+def main(directory='scripts'):
```

Comment:
this change is for future use, when we create examples and we want to run `python manage.py run examples/pong` and it gets the pong directory and run pong game

Reply:
Perfect!